docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.3"
+services:
+  postgres:
+    image: quay.io/azavea/postgis:2.2-postgres9.5-slim
+    environment:
+      - POSTGRES_DB=district_builder
+      - POSTGRES_USER=district_builder
+      - POSTGRES_PASSWORD=district_builder
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 15s
+      timeout: 5s
+      retries: 3
+
+  redis:
+    image: redis:3.2.11-alpine
```

Comment:
I think that we can shorten this to `3.2-alpine`, although, would be great to get rid of Redis in the future if we can.

Reply:
There's no need to use redis here I think, I just did because the project used redis previously. Do you mean it would be good to switch to memcached, or it would be good not to have to depend on a cache at all?

Reply:
We're actually not using Redis as a cache, but rather as key/value storage for adjacency information, which is used in a single, and rarely used, calculator. If we can find an alternate efficient way to perform that calculation, then Redis can be eliminated as a dependency. At this stage, we can probably just ignore dealing with Redis completely if we just don't use that calculator in any testing.

Reply:
My comment was mostly from the perspective of eliminating an architectural component. I'm cool with keeping it out for now based on Kenny's explanation.